hw/ip/prim/lint/prim_onehot_check.vlt

```diff
@@ -16,3 +16,7 @@ split_var -module "prim_onehot_check" -var "err_tree"
 // The clock and reset are only used for assertions in this module.
 lint_off -rule UNUSED -file "*/rtl/prim_onehot_check.sv" -match "Signal is not used: 'clk_i'"
 lint_off -rule UNUSED -file "*/rtl/prim_onehot_check.sv" -match "Signal is not used: 'rst_ni'"
+
+// The EnableAlertTriggerSVA parameter is only used by modules bound
+// in for DV testing. Waive the warning that we don't read it.
+lint_off -rule UNUSED -file "*/rtl/prim_double_lfsr.sv" -match "*EnableAlertTriggerSVA*"
```

Comment:
@msfschaffner I have this CI error. I added this waiver like the other prim, but it doesn't solve the error. Do you know how to fix this?
> ERROR: %Warning-UNUSED: ../src/lowrisc_prim_onehot_check_0/rtl/prim_onehot_check.sv:34:17: Parameter is not used: 'EnableAlertTriggerSVA'
>                                                                                   : ... In instance otbn_top_sim.u_otbn_core.u_otbn_rf_bignum.gen_rf_bignum_ff.u_otbn_rf_bignum_inner.u_prim_onehot_check
>    34 |   parameter bit EnableAlertTriggerSVA = 1
>       |                 ^~~~~~~~~~~~~~~~~~~~~
>                  ... For warning description see https://verilator.org/warn/UNUSED?v=4.210
>                  ... Use "/* verilator lint_off UNUSED */" and lint_on around source to disable this message.
>

Reply:
@weicaiyang do you need to update the file also? your rule seems to still say `prim_double_lfsr` right now.

Reply:
oh, I'm blind. didn't see this. Let me fix it